README.md

```diff
@@ -12,7 +12,7 @@
   </head>
   ```
 2. Add the class `animated` to the element you want to animate.
- You may also want to include the class `infinite` for an infinite loop.
+ You may also want to include the class `infinite` for an infinite loop, and if the element has no width, add the `centerAnimation` class to it.
```

Comment:
Again, the CSS for this class depends on a lot of other styles for it to work as expected, so remove this from the readme.